src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -408,8 +410,9 @@ abstract class DatabaseTest {
   protected val plantingZonesDao: PlantingZonesDao by lazyDao()
   protected val projectDocumentSettingsDao: ProjectDocumentSettingsDao by lazyDao()
   protected val projectReportSettingsDao: ProjectReportSettingsDao by lazyDao()
-  protected val projectsDao: ProjectsDao by lazyDao()
   protected val projectVotesDao: ProjectVotesDao by lazyDao()
+  protected val projectsDao: ProjectsDao by lazyDao()
+  protected val projectVoteDecisionDao: ProjectVoteDecisionsDao by lazyDao()
```

Comment:
This still seems out of order to me ("s" is before "v" and "d" is before "s").

Reply:
[![facepalm](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/WjlTemxb6oA4PgZFmj08/90847050-76a2-4df0-ad36-97675936b7e2/facepalm.gif)](https://app.graphite.dev/settings/meme-library?org=terraware)